_includes/v3.9/release-notes/v3.9.4-release-notes.md

```diff
@@ -0,0 +1,11 @@
+06 Dec 2019
+
+#### Bug fixes
+
+ - Don't remove IPv6 link-local addresses [felix #2175](https://github.com/projectcalico/felix/pull/2175) (@neiljerram)
```

Comment:
Did we not backport this IPv6 fix to v3.10 as well?

Reply:
Thanks, it was but was in a previous milestone.